src/runtime/gc_globals.go

```diff
@@ -1,4 +1,4 @@
-//go:build (gc.conservative || gc.precise) && (baremetal || tinygo.wasm)
+//go:build (gc.conservative || gc.precise) && (baremetal || tinygo.wasm) && !uefi
```

Comment:
I would suggest using the build tag `baremetal` for UEFI, because that's basically what it is (at least how I understand it).

Reply:
FindGlobals needs to be reimplemented for UEFI.
I just pushed the code that implements findGlobals.

Reply:
You can still implement this as a baremetal target. The only file you have to modify is src/runtime/gc_globals.go to make sure it doesn't compile for uefi, and add a new `findGlobals` just for UEFI (that then calls shared Windows code).